capture/crawler_folha.py

```diff
@@ -1,9 +1,140 @@
+                               %(levelname)s - %(message)s')
+
+# add formatter to stream_handler
+stream_handler.setFormatter(formatter)
+file_handler.setFormatter(formatter)
+
+# add stream_handler to logger
+logger.addHandler(stream_handler)  # uncomment for console output of messages
+logger.addHandler(file_handler)
+
+client = pymongo.MongoClient(settings.MONGOHOST, 27017)
+mcdb = client.MCDB
+ARTICLES = mcdb.articles  # Article Collection
+ARTICLES.ensure_index("source")
+
+
+def find_articles(page=None):
+    """
+    """
+
+    base_url = "http://www1.folha.uol.com.br/ultimas-noticias/"
+    if page is None:
+        index_url = base_url + "index.shtml"
+    else:
+        index_url = base_url + "noticias-{0}.shtml".format(page)
+
+    index = requests.get(index_url).content
+    soup = BeautifulSoup(index)
+    news_index = soup.find(**{'class': 'news-index'}).find('ol')
+    news_urls = [url.attrs['href'] for url in news_index.find_all('a')]
+    return news_urls
+
+def extract_title(article):
+    """ Extract the news title.
+    """
+
+    try:
+        title = article.title
+    except Exception as ex:
+        template = "An exception of type {0} occured during extraction of news title. Arguments:\n{1!r}"
+        message = template.format(type(ex).__name__, ex.args)
+        logger.exception(message)
+        return None
+    if title is None:
+        logger.error("The news title is None")
+    return title
+
+def extract_category(url):
+    """
+    """
+    try:
+        #returns 'http: or 'https:'
+        app_protocol = re.search('(.+?)/', url).group(1)
+        base_url = re.search(app_protocol + '//(.+?)/', url).group(1)
+        category = re.search(base_url + '/(.+?)/', url).group(1)
+    except AttributeError:
+        logger.error("Some problem has occured during extraction of article's category")
+        return None
+    return category
+
+def extract_content(article):
+    content_body = article.cleaned_text
+
+    if "post completo no blog" not in content_body:
+        return content_body
+    else:
+        logger.error("The article is into blog's link")
+        return None
+
+def download_article(url):
+    """ Download the html content of a news page
+
+    :param url: news page's url
+    :type url: string
+    :return: news page's content
+    :rtype: requests.models.Response
+    """
+
+    article = { 'link': url, 'source': 'crawler_folha_sao_paulo' }
+    logger.info("Downloading article: {0}".format(url))
+
+    try:
+        response = requests.get(url, timeout=30)
+    except ConnectionError:
+        logger.error("Failed to fetch:{0}".format(url))
+        return None
+    except Timeout:
+        logger.error("Timed out while fetching {0}".format(url))
+        return None
+
+    extractor = Goose({'use_meta_language': False, 'target_language':'pt'})
+    news = extractor.extract(url=url)
+
+    article['link_content'] = compress_content(response_content)
```

Comment:
It should be response.content, but is better to use response.text as you have already suggested.